index.html

```diff
@@ -54,9 +54,203 @@ <h2 class="supporting-text">
+                        <br> personalized reads; no accounts or
+                        <br> sign-ups required. has been the 
+                        <br>industry's standard
+                    </p>
+                    <section class="code-links2">
+                        <a class="css-code2" href="">html</a>
+                        <a class="css-code2" href="">bootstrap</a>
+                        <a class="css-code2" href="">Ruby</a>
+                    </section>
+                </div>
+                <a class="see-project3" href="">See Project</a>
+            </section>
+            <section class="card6">
+                <div class="imgPlaceholder2 ">
+                    <h1 class="post-title2">
+                        Profesional Art Printing Data
+                    </h1>
+                    <p class="suppoting-text3">
+                        A daily selection of privately
+                        <br> personalized reads; no accounts or
+                        <br> sign-ups required. has been the 
+                        <br>industry's standard
+                    </p>
+                    <section class="code-links2">
+                        <a class="css-code2" href="">html</a>
+                        <a class="css-code2" href="">bootstrap</a>
+                        <a class="css-code2" href="">Ruby</a>
+                    </section>
+                </div>
+                <a class="see-project3" href="">See Project</a>
+            </section>
+            
+        </section>
+    </section>
+</section>
+<section class="last-section">
+    <section class="about-me">
+        <h1 class="text1 pad">About me</h1>
+        <p class="paragragh1">Hello I’m a software developer! I can
+            <br> help you build a product , feature or
+            <br> website Look through some of my work 
+            <br> and experience! If you like what you see
+            <br> and have a project you need coded, don’t
+            <br> hestiate to contact me.</p>
+            <a class="resume" href="">Get My Resume</a>
+    </section>
+    <br><br><br><br><br><br>
+    <hr>
+    <section class="sec2">
+        <ul class="footer">
+            <li class="class1">
+                Language 
+            <ul class="language">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+            </li>
+            <li class="class1">Frameworks
+                <ul class="language">
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                    
+                </ul>
+            </li>
+            <li class="class1 skills"> Skills
+                <ul class="language">
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
```

Comment:
- [ ] KIndly ensure that the bullet shapes are properly aligned with the text to match the Figma design👍.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/177019758-2b8da50b-4497-44d8-954a-c603cecf224b.png) |![image](https://user-images.githubusercontent.com/63473584/177019772-02062e13-8771-42e0-a7e6-0d8bd88b3002.png)  |